src/Microsoft.Azure.SignalRService/HubInfo.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.SignalRService
+{
+    internal class HubInfo
```

Comment:
What's the purpose of having a class with only one property? Do we plan to add more properties in future?

Reply:
For now, nothing else to add. I delete the class. We can make a class if we have several data to pass.